### PR TITLE
Update the security policy

### PR DESCRIPTION
**Pull request by bmiklautz**
This commit updates the security with more details on the procedure  and also add GitHub's security advisory page as possibility to report a vulnerability.

### CODE REVIEW

**Review by akallabeth — APPROVED** [review]
LGTM

File: SECURITY.md [hunk 1]
```diff
@@ -1,20 +1,61 @@
...
-| master  | :white_check_mark: |
-| 2.x.x   | :white_check_mark: |
-| < 2.0.0 | :x:                |
 
-## Reporting a Vulnerability
+## Supported versions
+
+Security is very important for us therefore we try to provide security updates and support for
+the latest stable version as well as for the development branch.
+Since our development branch is, like the protcol itself, a moving target we won't request CVEs for issues that are *only* found on the development branch.
```
Comment (hardening) [inline SECURITY.md:15]: s/protcol/protocol/

File: SECURITY.md [hunk 2]
```diff
@@ -1,20 +1,61 @@
...
+
+## Reporting a vulnerability
+
+**IMPORTANT**: Please, do not file security vulnerabilities as public issues on GitHub
+
+In advance: **Thank you** for reporting a security vulnerability and making FreeRDP more stable! We really appreciate your effort.
+Please let us know who we should give the credit or attributions to.
+
+
+If you have found a security vulnerability in FreeRDP you can either direclty open an [Advisory on GitHub](https://github.com/FreeRDP/FreeRDP/security/advisories/new)[^1] or send us an email to mailto:security@freerdp.com
```
Comment (akallabeth) [inline SECURITY.md:34]: `s/direclty/directly/g`

File: SECURITY.md [hunk 3]
```diff
@@ -1,20 +1,61 @@
...
+
+
+## Disclosure procedure
+
+When the FreeRDP team receives a report one of the team members will be assigned as primary contact.
+The primary contact will do all further communications and coordinate the fix and release process. 
+
+How your report will be handled:
+
+* When a report is received we will acknolege the reception and review the reported issue(s) as soon as possible.
```
Comment (hardening) [inline SECURITY.md:50]: s/acknolege/acknowledge/

**Review by hardening — APPROVED** [review]
Except the typos LGTM

File: SECURITY.md (hunk 3 above)
Comment (akallabeth) [inline SECURITY.md:50]: `/s/acknolege/acknowlege/g`

**Comment by bmiklautz** [comment]
@hardening @akallabeth thanks for the review!

**Pull request merged**